docs/UserGuide.md

```diff
@@ -92,14 +93,33 @@ Here are some icons that will be used in this User Guide:
 * Words within `(parentheses)` are optional user inputs.
   e.g. in `list (/urgent)`, `/urgent` is an optional parameter.
 * Words in [blue]() are clickable links.
-* Within a code segment, `>>>` denotes the command typed by the user.
-    Example:  
-    ```
-    >>> select /sChapter 1
-        now in notebook section: Chapter 1
-    ```
+* Within a code segment, the symbols before the `$` symbol indicates the current mode and notebook/section the user is in.
+    * The following symbols represents the respective address:
+        * `T:$` - The user is in TimeTable mode.  
+        * `N:$` - The user is in Notebook mode.
+        * `N/[NOTEBOOK NAME]` - The user is in Notebook mode, and is in the indicated notebook with 'NOTEBOOK NAME'.
+        * `N/[NOTEBOOK NAME]/[SECTION NAME]` - The user is in Notebook mode, and is in the indicated section with 'SECTION NAME' of the notebook with 'NOTEBOOK NAME'.  
```

Comment:
It displays even up to N/[NOTEBOOK NAME]/[SECTION NAME]/[PAGE NAME] actually...

Reply:
OH LOL
what the hek HAHHAHA ok i go edit